packages/tasks/src/model-libraries-snippets.ts

```diff
@@ -401,6 +401,23 @@ export const timm = (model: ModelData): string[] => [
 model = timm.create_model("hf_hub:${model.id}", pretrained=True)`,
 ];
 
+export const seed_story = (): string[] => [
+	`# Install from https://github.com/TencentARC/SEED-Story.git
+# Use in SEED-Story (https://huggingface.co/TencentARC/SEED-Story)
+# seed_story_cfg_path refers to 'https://github.com/TencentARC/SEED-Story/blob/master/configs/clm_models/agent_7b_sft.yaml'
+# llm_cfg_path refers to 'https://github.com/TencentARC/SEED-Story/blob/master/configs/clm_models/llama2chat7b_lora.yaml'
```

Comment:
Fine for me to merge with this, but I feel these could be hosted under the HF repository, and then `OmegaConf.load("TencentARC/SEED-Story")` could load these configs directly from there